Training5/thongnq-training5-task2/src/pages/Auth/index.js

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react'
+
+import { Wrapper, Content, Form } from './Auth.styles';
+
+import {Redirect, useHistory} from 'react-router-dom';
+
+import {useDispatch, useSelector} from "react-redux";
+import {login} from "../../state/actions/auth";
+
+const Auth = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const history = useHistory();
+
+
+    const dispatch = useDispatch();
+    const auth = useSelector(state => state.auth);
+    console.log(auth);
+
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(login(email, password))
+        .then(() => {
+            if(auth.user){
+                history.push('/app');
+                window.location.reload();
```

Comment:
@Thong-Nguyen-Zigvy 
why we need to use window.location.reload here, it's bad practise